dbt_subprojects/hourly_spellbook/models/_project/safe/linea/safe_linea_schema.yml

```diff
@@ -0,0 +1,143 @@
+version: 2
+
+models:
+  - name: safe_linea_safes
+    meta:
+      blockchain: linea
+      project: safe
+      contributors: danielpartida
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
```

Comment:
```suggestion

```

Reply:
safe to remove freshness.

plz add unique combo of columns test instead, as you did in scroll PR. ensure keys match model unique keys.
this applies to any incremental model in this schema file.